config/routes.rb

```diff
@@ -11,5 +11,8 @@
 
```

Comment:
- [ ]  It would be good if you could set the correct root according to the changes in the project.

```
root to: 'public_recipes#index',
```